packages/astro/test/astro-directives.test.js

```diff
@@ -36,8 +36,6 @@ describe('Directives', async () => {
 		const html = await fixture.readFile('/define-vars/index.html');
 		const $ = cheerio.load(html);
 
-		expect($('style')).to.have.lengthOf(2);
```

Comment:
Should this check for a length of 1 instead?

Reply:
New behavior is that `define:vars` won't generate any inline styles, so [6d6352f](https://github.com/withastro/astro/pull/5339/commits/6d6352f04f67aa07501ea26ae40b018a9fa68462) verifies that with `expect($('style')).to.have.lengthOf(0)`

Reply:
Ah, ok cool